### PR TITLE
Lower case usernames and 403 page

### PR DESCRIPTION
**Pull request by astiunov**
Added `normalize_username` to authenticators (should solve #33) and simple 403 page for users, that are not in a whitelist.

### CODE REVIEW

File: share/static/html/403.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <h1 style="font-size: 55px">everware</h1>
+    <p style="font-size: 26px">Sharing is caring</p>
+</header>
+
+<div class="container">
+  <div class="row row-centered">
+    <div class="col-sm-8">
+      <h1> 403 Forbidden </h1>
+      <p>
+      You are not in a whitelist of this server. Please, contact to system administator to gain access.
```
Comment (betatim) [inline share/static/html/403.html]: "Your username is not whitelisted on this server. Please contact the system administrator."

**Comment by betatim** [comment]
Can you add a test with a `UserName`, a `username` and `notinwhitelistgets403`. Checkout [`test_pages.py`](https://github.com/everware/everware/blob/master/everware/tests/test_pages.py) where the github part is mocked. We should be able to make it work so we can test this. I'd probably start a new `test_auth.py`

**Comment by astiunov** [comment]
But I need to interact with github (i.e. use github authenticator) to check it. Is it bad?

**Comment by anaderi** [comment]
@astiunov you don't have to interact with github, you just have to mock it. 
see for example this post on mocking github API
https://www.appneta.com/blog/python-unit-test-mock/

**Comment by betatim** [comment]
The tests I linked to mock the oauth part already, so you can use that no?

**Comment by anaderi** [comment]
ah, haven't realized that

**Comment by betatim** [comment]
To avoid confusion, I was referring to what is in `everware/tests`. Right now it might not do exactly what is needed to test lower/upper case and a whitelist but we can build on it.

**Comment by betatim** [comment]
Nice work!

**Pull request merged**